gamehype/routes.py

```diff
@@ -1,6 +1,18 @@
+from flask import render_template
 from gamehype import app
 
 @app.route('/')
 @app.route('/index')
 def index():
-    return "Hello, Idiot! You did it!"
+    user = {'username': 'Timmy Jimmy'}
+    ratings = [
+        {
+            'hypeman': {'username': 'Matt'},
```

Comment:
Looks good.  We'll need to change hypeman/hype to owner/rating to match the DB (or viceversa) but that can come later